app/components/certificate/details/certificate-details.component.ts

```diff
@@ -0,0 +1,108 @@
+import { ChangeDetectionStrategy, Component, OnDestroy, OnInit } from "@angular/core";
+import { MatDialog, MatDialogConfig } from "@angular/material";
+import { ActivatedRoute, Router } from "@angular/router";
+import { autobind } from "@batch-flask/core";
+import { ElectronRemote } from "@batch-flask/ui";
+import { Observable, Subscription } from "rxjs";
```

Comment:
move flask imports out of this block

Reply:
i think they belong here

Reply:
ok cool. if you are happy then i am also :)